docs/src/ci.md

```diff
@@ -51,8 +51,16 @@ steps:
       node-version: '14'
   - name: Install operating system dependencies
     run: npx playwright install-deps
+  - name: Install dependencies
```

Comment:
this step should go before `Install operating system dependencies`: this way you'll use the Playwright binary version that your project depends on.